.cirrus.yml

```
@@ -0,0 +1,9 @@
+task:
+  freebsd_instance:
+    matrix:
+      - image: freebsd-11-2-release-amd64
+      - image: freebsd-12-0-release-amd64
+  install_script: pkg install -y gmake
+  script:
+    - gmake all test_programs && cat benchmark | ./benchmark &&  ./test_slow_decompression
```

Maybe improve readability?

```suggestion
    - gmake all test_programs
    - cat benchmark | ./benchmark
    - ./test_slow_decompression
```